src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.tsx

```diff
@@ -0,0 +1,452 @@
+        tableQuery = `${createTableQueryPrefix} WITH ${autoPilotCommand}=${throughput};`;
+      } else {
+        tableQuery = `${createTableQueryPrefix} WITH cosmosdb_provisioned_throughput=${throughput};`;
+      }
+    } else {
+      tableQuery = `${createTableQueryPrefix};`;
+    }
+
+    const addCollectionPaneStartMessage = {
+      collection: {
+        id: tableId,
+        storage: Constants.BackendDefaults.multiPartitionStorageInGb,
+        offerThroughput: throughput,
+        partitionKey: "",
+        databaseId: keyspaceId,
+        hasDedicatedThroughput: dedicateTableThroughput,
+      },
+      keyspaceHasSharedOffer: keyspaceHasSharedOffer,
+      subscriptionType: userContext.subscriptionType,
+      subscriptionQuotaId: userContext.quotaId,
+      defaultsCheck: {
+        storage: "u",
+        throughput: throughput,
+        flight: userContext.addCollectionFlight,
+      },
+      dataExplorerArea: Constants.Areas.ContextualPane,
+      toCreateKeyspace: toCreateKeyspace,
+      createKeyspaceQuery: createKeyspaceQuery,
+      createTableQuery: tableQuery,
+    };
+
+    const startKey: number = TelemetryProcessor.traceStart(Action.CreateCollection, addCollectionPaneStartMessage);
+    try {
+      if (toCreateKeyspace) {
+        await cassandraApiClient.createTableAndKeyspace(
+          userContext?.databaseAccount?.properties?.cassandraEndpoint,
+          userContext?.databaseAccount?.id,
+          container,
+          tableQuery,
+          createKeyspaceQuery
+        );
+      } else {
+        await cassandraApiClient.createTableAndKeyspace(
+          userContext?.databaseAccount?.properties?.cassandraEndpoint,
+          userContext?.databaseAccount?.id,
+          container,
+          tableQuery
+        );
+      }
+      container.refreshAllDatabases();
+      setIsExecuting(false);
+      closePanel();
+
+      TelemetryProcessor.traceSuccess(Action.CreateCollection, addCollectionPaneStartMessage, startKey);
+    } catch (error) {
+      const errorMessage = getErrorMessage(error);
+      setFormErrors(errorMessage);
+      setIsExecuting(false);
+      const addCollectionPaneFailedMessage = {
+        ...addCollectionPaneStartMessage,
+        error: errorMessage,
+        errorStack: getErrorStack(error),
+      };
+      TelemetryProcessor.traceFailure(Action.CreateCollection, addCollectionPaneFailedMessage, startKey);
+    }
+  };
+  const handleOnChangeKeyspaceType = (ev: React.FormEvent<HTMLInputElement>, mode: string): void => {
+    setKeyspaceCreateNew(mode === "Create new");
+  };
+
+  const genericPaneProps: GenericRightPaneProps = {
+    expandConsole: () => container.expandConsole(),
+    formError: formErrors,
+    formErrorDetail: "",
+    id: "cassandraaddcollectionpane",
+    isExecuting,
+    title: "Add Table",
+    submitButtonText: "Apply",
+    onClose: closePanel,
+    onSubmit: submit,
+  };
+  return (
+    <GenericRightPaneComponent {...genericPaneProps}>
```

Comment:
Remove `GenericRightPaneComponent`. I believe we stopped using this in the other panels correct?

Reply:
done. yes correct